cub/cub/device/dispatch/kernels/transform.cuh

```diff
@@ -640,6 +648,9 @@ _CCCL_DEVICE void transform_kernel_ublkcp(
 
       // TODO(ahendriksen): this could only have ptx::sem_relaxed, but this is not available yet
       ptx::mbarrier_arrive_expect_tx(ptx::sem_release, ptx::scope_cta, ptx::space_shared, &bar, total_copied);
+
+      while (!ptx::mbarrier_try_wait_parity(&bar, 0))
+        ;
```

Comment:
Important: this should not be needed, since we have the same `mbarrier_try_wait_parity` after the `if-else` already (line 697).